docs/task_file_format_summarization.md

```diff
@@ -0,0 +1,59 @@
+# File Formats Supported by Different Tasks
```

Comment:
Regarding the file name: "summarization" is an NLP task so I was confused when I saw the file name here. Could we just call it `task_file_formats.md`?